lib/src/main/java/com/thenewboston/data/dto/bankapi/invalidblockdto/request/PostInvalidBlockRequest.kt

```diff
@@ -0,0 +1,63 @@
+package com.thenewboston.data.dto.bankapi.invalidblockdto.request
+
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class PostInvalidBlockRequest(
+
+    @SerialName("message")
+    val message: PostInvalidBlockMessage,
```

Comment:
For this case we could name this `InvalidBlockMessage` Not sure why the API has 2 message nodes which would make it confusing but not much we can about for it now.